emulsify.drush.inc

```diff
@@ -0,0 +1,299 @@
+  if (drush_get_option('slim') === TRUE) {
+    return $slim_array;
+  }
+  else {
+    return array_merge($slim_array, array(
+      'emulsify.breakpoints.yml',
+      'emulsify.libraries.yml',
+    ));
+  }
+}
+
+/**
+ * Alter strings within files.
+ */
+function _emulsify_alter_files($theme_path, $files_to_alter = array(), $alterations = array(), $absolute = FALSE) {
+  if (empty($files_to_alter) || empty($alterations)) {
+    return TRUE;
+  }
+  foreach ($files_to_alter as $file_to_replace) {
+    if ($absolute === TRUE) {
+      $file_type = filetype($file_to_replace);
+      $file_path = $file_to_replace;
+    }
+    else {
+      $file_type = filetype($theme_path . DIRECTORY_SEPARATOR . $file_to_replace);
+      $file_path = $theme_path . DIRECTORY_SEPARATOR . $file_to_replace;
+    }
+
+    if ($file_type === 'dir') {
+      $files = file_scan_directory($file_path, '/\.*/');
+      _emulsify_alter_files($theme_path, array_keys($files), $alterations, TRUE);
+    }
+    elseif ($file_type === 'file') {
+      _emulsify_file_str_replace($file_path, array_keys($alterations), $alterations);
+    }
+  }
+}
+
+/**
+ * Copy files.
+ *
+ * @param array $files
+ *   An array of files (strings) to copy.
+ * @param string $destination_path
+ *   A string representing the destination path.
+ *
+ * @return bool
+ *   A boolean representing the success or failure of the command.
+ */
+function _emulsify_copy_files($files = array(), $destination_path = '') {
+  if (empty($files) || empty($destination_path)) {
+    return FALSE;
+  }
+  file_prepare_directory($destination_path, FILE_CREATE_DIRECTORY);
+  foreach ($files as $files_to_copy) {
+    drush_copy_dir(__DIR__ . DIRECTORY_SEPARATOR . $files_to_copy, $destination_path . DIRECTORY_SEPARATOR . $files_to_copy);
+  }
+  return TRUE;
+}
+
+/**
+ * Rename files.
+ */
+function _emulsify_rename_files($theme_path, $machine_name, $files_to_rename = array()) {
+  foreach ($files_to_rename as $file_to_rename_path) {
+    $file_original_path = $theme_path . DIRECTORY_SEPARATOR . $file_to_rename_path;
+    $file_new_path = $theme_path . DIRECTORY_SEPARATOR . str_replace('emulsify', $machine_name, $file_to_rename_path);
+    drush_op('rename', drush_normalize_path($file_original_path), drush_normalize_path($file_new_path));
+  }
+}
+
+/**
+ * Replace strings in a file.
+ */
+function _emulsify_file_str_replace($file_path, $find, $replace) {
+  $file_path = drush_normalize_path($file_path);
+  $file_contents = file_get_contents($file_path);
+  $file_contents = str_replace($find, $replace, $file_contents);
+  file_put_contents($file_path, $file_contents);
+}
+
+/**
+ * Notifies the user of success.
+ */
+function _emulsify_notify($name, $theme_path) {
+  // Notify user of the newly created theme.
+  $message = 'Successfully created the Emulsify theme "!name" created in: !path, you can now run \'yarn\' or \'npm install\' to install the node modules.';
```

Comment:
`yarn` and `yarn install` are both valid ways to install modules. (I'm fine with adding "install", just wanted to mention this.)